docs/content/preview/yugabyte-platform/administer-yugabyte-platform/back-up-restore-k8s.md

```diff
@@ -27,54 +27,59 @@ type: docs
   </li>
 </ul>
 
-<br>YugabyteDB Anywhere installations include configuration settings, certificates and keys, and other components required for creating and managing YugabyteDB universes.
+YugabyteDB Anywhere installations include configuration settings, certificates and keys, and other components required for creating and managing YugabyteDB universes.
 
-You can use the YugabyteDB Anywhere backup script to back up an existing YugabyteDB Anywhere server and restore it, when needed, for disaster recovery or migrating to a new server.
+You can use the YugabyteDB Anywhere `yb_platform_backup.sh` script to back up an existing YugabyteDB Anywhere server and restore it, when needed, for disaster recovery or migrating to a new server.
 
 {{< note title="Note" >}}
 
 You cannot back up and restore Prometheus metrics data.
 
 {{< /note >}}
 
-The YugabyteDB Anywhere UI is used for creating and managing YugabyteDB universes, or clusters, on one or more regions (across public cloud and private on-premises data centers). For details, see [YugabyteDB Anywhere overview](../../../yugabyte-platform/).
+## Download the script
 
-## Back up a YugabyteDB Anywhere server
+Download the version of the backup script that corresponds to the version of YugabyteDB Anywhere that you are backing up and restoring.
 
-You can back up the YugabyteDB Anywhere server as follows:
+For example, you can copy the `yb_platform_backup.sh` script from the `yugabyte-db` repository using the following `wget` command:
 
-- Copy the YugabyteDB Anywhere backup script `yb_platform_backup.sh` from the `yugabyte-db` repository to your local workstation using the following wget command:
+```sh
+wget https://raw.githubusercontent.com/yugabyte/yugabyte-db/{{< yb-version version="preview" format="short">}}/managed/devops/bin/yb_platform_backup.sh
```

Comment:
Unfortunately, I don't think this approach will address the problem with the docs linking to branches instead of tags. 

Not every release has a corresponding branch, since our current development workflow only creates a branch when we need to build a hotfix on top of it. As an example, if a customer is running a hypothetical 2.18.2.0 release, when they try to run this wget to retrieve the corresponding backup script from the 2.18.2 branch, they may receive a 404 error because the branch would only be created if and when a 2.18.2.1 hotfix was created.

The approach recommended to me by the YBA team is to use the specific tag corresponding with the release version. For the example above, this would be v2.18.2.0, making the full path `https://raw.githubusercontent.com/yugabyte/yugabyte-db/v2.18.2.0/managed/devops/bin/yb_platform_backup.sh`